content/cloud/serverless-ai.md

```diff
@@ -0,0 +1,72 @@
+title = "Serverless AI"
+template = "cloud_main"
+date = "2023-09-05T09:00:00Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/cloud/serverless-ai.md"
+
+---
+- [Accessing Private Beta](#accessing-private-beta)
+- [Quotas And Service Limitations For Fermyon Serverless AI](#quotas-and-service-limitations-for-fermyon-serverless-ai)
+- [FAQ](#faq)
+- [Next Steps](#next-steps)
+
+## Accessing Private Beta
+
+The Fermyon Serverless AI is currently in private beta. To request access to the private beta, please fill out this short [sign-up form](https://fibsu0jcu2g.typeform.com/to/mNzgXRvB#hubspot_utk=xxxxx&hubspot_page_name=xxxxx&hubspot_page_url=xxxxx).
+ 
+> Please note that the private beta is limited in space, and all requests cannot be guaranteed. 
+
+Once you have access to the private beta, please ensure you have the `canary` version of Spin installed:
+
+<!-- @selectiveCpy -->
+
+```bash
+$ curl -fsSL https://developer.fermyon.com/downloads/install.sh | bash -s -- -v canary
+```
+
+Also, for TypeScript/JavaScript examples, please ensure you have the latest TypeScript/JavaScript SDK installed:
```

Comment:
Do we also need the latest plugin for js2wasm?

`spin plugin update & spin plugin upgrade js2wasm`

Reply:
Not if we let then use the installer, it will get it for them.